From d0dd1ef31e441a2055fe431f86dcb23a5d336632 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Wed, 31 Jan 2018 17:31:33 +0800
Subject: [PATCH 3405/5242] MLK-17480: hdp: Change dp_vic to int variable

commit  51ab3b3c7027b7fd0a84c54b846ebe9d171e95f5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change dp_vic to int variable.
sync data type with the return variable of function imx_get_vic_index.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
(cherry picked from commit bb5bdf30ec7fc61c918e0a956d761e17d79f96ab)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdp.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdp.c b/drivers/gpu/drm/imx/hdp/imx-hdp.c
index 829de39..5bb43a5 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdp.c
@@ -628,7 +628,7 @@ static int imx_hdp_connector_get_modes(struct drm_connector *connector)
 					     connector);
 	enum drm_mode_status mode_status = MODE_OK;
 	struct drm_cmdline_mode *cmdline_mode;
-	u32 hdp_vic;
+	int hdp_vic;
 
 	cmdline_mode = &connector->cmdline_mode;
 
-- 
1.7.9.5

